app/assets/javascripts/admin/models/email-style.js.es6

```diff
@@ -0,0 +1,27 @@
+import { ajax } from "discourse/lib/ajax";
+import RestModel from "discourse/models/rest";
+
+const EmailStyle = RestModel.extend({
+  changed: false,
+
+  setField(fieldName, value) {
+    this.set(`${fieldName}`, value);
+    this.set("changed", true);
+  },
+
+  saveChanges() {
```

Comment:
Since you're using a `RestModel` you probably don't need to implement this method, and could install call `model.update(...)` instead of `model.saveChanges()`